!!python/object:__main__.CurrentMap
map_dir: /home/mkac/catkin_ws/src/agv_lidar/agvlidar_navigation/maps/
map_file: office3
